gui/wxpython/tplot/g.gui.tplot.html

```diff
@@ -18,7 +18,13 @@ <h2>DESCRIPTION</h2>
   <li>zoom and pan,</li>
   <li>change labels to x and y axes,</li>
   <li>add title to the plot, and</li>
-  <li>export the time series values to a CSV file</li>
+  <li>export the time series values to a CSV file (x axis data has date time string format,
+    if you want to use for calculating simple regression model in the
+    <a href="https://www.r-project.org/">R environment<a>,
+    <a href="https://www.libreoffice.org/">LibreOffice</a>
+    etc., you get different calculated formula <pre>y = a + b*x</pre> because they use
+    a reference date other than the UNIX Epoch time (00:00:00 UTC on 1 January 1970)).
```

Comment:
```suggestion
    etc., you will obtain a different calculated formula <pre>y = a + b*x</pre> because these
    software packages use a reference date other than the UNIX Epoch time (00:00:00 UTC on 1 January 1970)).
```